configs/openmrs/initializer_config/ampathforms/prenatal_form.json

```diff
@@ -27,16 +28,20 @@
                 "disallowDecimals": true,
                 "step": 1,
                 "concept": "d682ced8-f3c1-4658-8645-a41bdb104893",
-                "min": "0",
-                "max": "15"
+                "min": 0,
+                "max": 15,
+                "defaultValue": 1
               },
               "id": "parity"
             },
             {
               "type": "obs",
               "questionOptions": {
                 "rendering": "text",
-                "concept": "87cbeb2a-030a-48da-beb2-ee58f84414c0"
+                "concept": "87cbeb2a-030a-48da-beb2-ee58f84414c0",
+                "calculate": {
+                  "calculateExpression": "'G' + parseInt(gravidity) + 'P' + parseFloat(parity)"
```

Comment:
❤️

Reply:
Could you try to make this read only?
Did you try the [fixed-value](https://openmrs.atlassian.net/wiki/spaces/docs/pages/68681754/Field+Types+Reference#fixed-value) render?